### PR TITLE
[DE] Update `Client/Beatmap_editor`, `Modding/Kudosu`, `Skinning`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
`Skinning` was not correctly marked as outdated but is now up-to-date.

- [x] Remove `no_native_review` in `Modding/Kudosu` if PR gets reviewed

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/de.md [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
+- Song-Setup
 
-- [Beatmapping](/wiki/Beatmapping) (Grundladen zum Mapping)
-- [Beat Snap Divisor](Beat_Snap_Divisor)
-- [Distance Snap](Distance_snap)
-- [Custom Sample Overrides](/wiki/Guides/Custom_Sample_Overrides)
-- [Kiai Time](/wiki/Gameplay/Kiai_time)
+## Compose
+
+Im Tab [Compose](/wiki/Client/Beatmap_editor/Compose) lassen sich alle [Hit-Objekte](/wiki/Hit_object) und andere gameplay-bezogene Elemente erstellen. Viele Mapper verbringen infolgedessen die meiste ihrer Zeit in diesem Bereich.
```
Comment (FamousPig) [inline wiki/Client/Beatmap_editor/de.md:18]:
```suggestion
Im Tab [Compose](/wiki/Client/Beatmap_editor/Compose) lassen sich alle [Hit-Objekte](/wiki/Hit_object) und andere Gameplay-bezogene Elemente erstellen. Viele Mapper verbringen infolgedessen die meiste ihrer Zeit in diesem Bereich.
```

File: wiki/Skinning/de.md [hunk 2]
```diff
@@ -4,11 +4,9 @@ Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Stan
...
+*Hauptseite: [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List)*
 
-Wenn du einen speziellen Skin in deiner Beatmap verwendest, wird dies als Beatmap-spezifisches Skinning bezeichnet. Hierbei werden die einzelnen Elemente des Skins direkt in den Beatmap-Ordner gepackt und dadurch im Spiel verwendet, sofern der Spieler dies nicht ausdrücklich anders eingestellt hat. Wenn du einen solchen Skin verwenden möchtest, ist das [Ranking Criteria](/wiki/Ranking_Criteria) sowie die [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List) zu beachten. 
-
-Falls dein Beatmap-Skin ein gameplay-relevantes Element enthält, müssen alle obligatorischen Elemente aus der zugehörigen Skinsetliste ebenfalls in deiner Beatmap enthalten sein. Genau genommen beziehen sich diese Einschränkungen nur auf Beatmap-Skins für Beatmaps, die den Ranking-Prozess durchlaufen, aber es ist als Skinner generell sinnvoll, diese Liste beim Erstellen eines Skins in Erwägung zu ziehen.
+**Nur für Beatmaps**: Wenn dein Beatmap-Skin ein einziges Element aus den Sets enthält, die in den Ranking-Kriterien aufgelistet werden, muss er auch alle anderen Elemente aus dem besagten Skin-Set enthalten.
```
Comment (FamousPig) [inline wiki/Skinning/de.md:9]: There is a sentence missing here.

 https://github.com/ppy/osu-wiki/blob/fac8d562ef9bd43494421e3b57db5b305b044694/wiki/Skinning/en.md?plain=1#L9

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Skinning/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Skinning/de.md:9]:
```suggestion
**Nur für Beatmaps**: Wenn dein Beatmap-Skin ein einziges Element aus den Sets enthält, die in den Ranking-Kriterien aufgelistet werden, muss er auch alle anderen Elemente aus dem besagten Skin-Set enthalten. Das trifft nur auf Beatmap-Skins zu, aber andere Skinersteller können diese Listen auch in Betracht ziehen.
```
Thanks for noticing 👍

File: wiki/Modding/Kudosu/de.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
 ---
 stub: true
 no_native_review: true
```
Comment (The-Last-Cookie) [inline wiki/Modding/Kudosu/de.md:3]:
```suggestion
```

**Review by FamousPig — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**